oshc/main/templates/registration/registration_form.html

```diff
@@ -61,23 +60,23 @@ <h1 class="form-title text-center">{% trans 'Sign Up Here' %}</h1>
         case 1:
             passwordStrength="Weak";
             color="red";
-            document.getElementById("abc").value=25;
+            document.getElementById("meter").value=25;
             break;
         case 2:
```

Comment:
@vaibhavsingh97  not empty cases password strength is weak when the check value is 0 and 1 similarly when check value is either 2 or 3 strength is good